meta/netbeansrelease.json

```diff
@@ -1338,6 +1338,38 @@
             "year": "2024"
         }
     },
+    "release230": {
+        "position": "36",
+        "ant": "ant_latest",
+        "jdk": "jdk_17_latest",
+        "jdktoolapidoc" : "jdk_22_latest",
+        "jdk_apidoc": "https://docs.oracle.com/en/java/javase/11/docs/api/",
```

Comment:
should this be 17 too?

Reply:
jdk_apidoc ? in fact not yet used it's still jdk 8 in the jdk link ( can be changed later maybe)
jdktoolapidoc ? it should be "nbjavac compatible" otherwise issue occurs (often in java.source)

Reply:
was thinking about `jdk_apidoc`, since release2299 has it at 17 already

Reply:
<!-- Docs to link to: -->
        <property name="javadoc.web.root" value="http://root/"/>
        <property name="javadoc.docs.jdk" value="https://docs.oracle.com/javase/8/docs/api"/>
        
        will adress this later I you don't mind. I also wanted to make a dropdown instead of the scrollable versions changer. Implies redoing "patch" on release branches. 
![image](https://github.com/user-attachments/assets/47d917e7-87d5-4501-b90d-ac85c4172b24)

Reply:
its fine with me if this is intended, just wanted to point it out that it is different than the rest. It also says 11 and not 8.